packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -65,20 +72,89 @@ const confirmDialogsPropTypes = {
   onToggleDialog: PropTypes.func.isRequired,
 };
 
+/* -------------------------------------------------------------------------------------------------
+ * BoldChunk
+ * -----------------------------------------------------------------------------------------------*/
+
+const BoldChunk = (chunks) => (
+  <Typography fontWeight="bold" data-testid="draft-rel-warning">
+    {chunks}
+  </Typography>
+);
+
 /* -------------------------------------------------------------------------------------------------
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+export const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+  const { selectedEntries } = useTableContext();
+  const toggleNotification = useNotification();
+  const { formatAPIError } = useAPIErrorHandler(getTrad);
+  const {
+    contentType: { uid: slug },
+  } = useSelector(listViewDomain());
+
+  const {
+    data: countDraftRelations,
+    isLoading,
+    isError,
+  } = useQuery(
+    ['content-manager', 'draft-relations', slug, selectedEntries],
+    async () => {
+      /**
+       * TODO: refactor the API call when the param serializer will be ready
+       */
```

Comment:
double check your PR for small mistakes like this please 😄

Reply:
yes, sorry, I removed the comment